docs/user_guides/single_table/constraints.rst

```diff
@@ -228,7 +249,8 @@ constraint by passing it:
     years_in_the_company_constraint = ColumnFormula(
         column='years_in_the_company',
         formula=years_in_the_company,
-        handling_strategy='transform'
+        handling_strategy='transform',
+        drop_column=False
```

Comment:
Since `CustomFormula` drops the column, we encounter a `KeyError: 'years_in_the_company'` when running the `GreaterThan` constraint; specifically when we look up the `dtype` of the column in the fit stage
https://github.com/sdv-dev/SDV/blob/505eac9f8c9bc6ce583163b201f9f08400f1b600/sdv/constraints/tabular.py#L331

We can swap the order of the constraints such that we apply `GreaterThan` then `ColumnFormula` in order to evade this situation.

Reply:
Yes, let's swap the order then